### PR TITLE
chore(RHINENG-9504): Update staleness dropdowns

### PR DESCRIPTION
**Pull request by adonispuente**
[RHINENG-9504](https://issues.redhat.com/browse/RHINENG-9504) 
This adds new items to 2 dropdowns in staleness and deletion. 
I will make a follow up PR updating the dropdown from pf4 to pf5 

System stale warning: add 2 days to 6 days - keep the rest (starting 7 days).
System deletion: add 3 days to 7 days - keep the rest (starting 14 days).

### CODE REVIEW

File: src/components/InventoryHostStaleness/constants.js [hunk 1]
```diff
@@ -405,15 +405,55 @@ export const systemStalenessItems = (activeTabKey) => {
 export const systemStalenessWarningItems = (activeTabKey) => {
   return [
```
Comment (bastilian) [inline src/components/InventoryHostStaleness/constants.js:406]: I think we can do these nicer. Maybe an array 

```js
const allDays = [2, 3, 4, 5, 6, 7, 14, 21, 30, 60, 90, 120];
const apiKey = activeTabKey ? 'immutable_time_to_stale_warning' : 'conventional_time_to_stale_warning';
const title = 'System stale warning';

allDays.map((value) => ({
       name: value + ' days',
       value,
       apiKey,
       title,
     })
```


Same below. :)

Comment (adonispuente) [reply src/components/InventoryHostStaleness/constants.js:406]: ahh nice implementation. cuts down the file size a bit, sendin this up!

File: src/components/InventoryHostStaleness/constants.js [hunk 2]
```diff
@@ -346,209 +346,57 @@ export const InventoryHostStalenessPopover = ({ hasEdgeSystems }) => {
...
+    ? 'immutable_time_to_stale'
+    : 'conventional_time_to_stale';
+  const title = 'System staleness';
+  const newItems = allDays.map((value) => ({
+    name: value + ' days',
+    value,
+    apiKey,
+    title,
+  }));
+  newItems[0].modalMessage =
```
[the remark this answers is not in the document]
Comment (bastilian) [reply src/components/InventoryHostStaleness/constants.js:360]: ... well, ok. One case I could imaging that it happens accidentally that `[0]` is not day `1` is if someone overlooked this and added some sorting to the `.map` above.

[the remark this answers is not in the document]
Comment (bastilian) [reply src/components/InventoryHostStaleness/constants.js:360]: ... I'm curious though, why this modalMessage should only be on the first... all of these items are staleness time amounts...

[the remark this answers is not in the document]
Comment (adonispuente) [reply src/components/InventoryHostStaleness/constants.js:360]: we can insert them into every pair, however the dropdowns only check for the first item currently. that can also be changed but both result in the same thing.

**Review by gkarat — APPROVED** [review]
LGTM! Thanks @adonispuente, I have run the PR locally and can see the dropdown are updated with new values. Haven't done any other checks/tests though.

**Pull request merged**